ghost/core/core/server/services/i18n.js

```
@@ -3,7 +3,37 @@ const debug = require('@tryghost/debug')('i18n');
+    return i18nInstance.loadLanguages(locale);
+}
+
+/**
+ * Temporarily changes the locale for a specific operation
+ * @param {string} locale - The locale to use
+ * @param {Function} operation - The operation to perform with the temporary locale
+ * @returns {Promise<any>} The result of the operation
+ */
+async function withLocale(locale, operation) {
+    if (!i18nInstance) {
+        throw new Error('i18n not initialized');
```

ESLint

[error] Direct calls to new Error() are not allowed. Please use @tryghost/errors.

(ghost/ghost-custom/no-native-error)